prow/cmd/deck/static/script.js

```diff
@@ -549,30 +549,37 @@ function createRerunCell(modal, rerun_command, prowjob) {
 
 function stateCell(state) {
     const c = document.createElement("td");
-    c.className = state;
+    c.classList.add(...["icon-cell"]);
 
     let displayState = "";
+    let displayIcon = "";
     switch (state) {
         case "pending":
             displayState = "Pending";
+            displayIcon = "access_time";
             break;
         case "success":
-            displayState = "Successful";
+            displayState = "Succeded";
+            displayIcon = "check_circle";
             break;
         case "failure":
             displayState = "Failed";
+            displayIcon = "error";
             break;
         case "aborted":
             displayState = "Aborted";
+            displayIcon = "clear";
             break;
         case "error":
             displayState = "Error";
+            displayIcon = "warning";
             break;
     }
-    const stateIndicator = document.createElement("DIV");
-    stateIndicator.classList.add(...["state", state]);
+    const stateIndicator = document.createElement("I");
+    stateIndicator.classList.add(...["material-icons", "state", state]);
```

Comment:
You dont need to unwrap the array, just

`...classList.add("material-icons", "state", state);`

I was my mistake there.

Reply:
fixed